instrumentation/ktor/ktor-1.0/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v1_0/KtorServerTracing.kt

```diff
@@ -75,6 +75,10 @@ class KtorServerTracing private constructor(
       httpAttributesExtractorBuilder.setCapturedResponseHeaders(responseHeaders)
     }
 
+    fun setKnownMethods(knownMethods: Set<String>) {
+      httpAttributesExtractorBuilder.setKnownMethods(knownMethods)
```

Comment:
I think it might be more idiomatic in kotlin to do this (assuming it compiles, lol):
```suggestion
      httpAttributesExtractorBuilder.knownMethods = knownMethods
```

Reply:
Unfortunately it doesn't work 😢 
```
Cannot access 'knownMethods': it is package-private in 'HttpServerAttributesExtractorBuilder'
```